backend/segmentation/models/microtubule/wrapper.py

```
@@ -161,10 +161,14 @@ def predict(self, image_np, seed_threshold: Optional[float] = None,
         embedding_samples: list = []
         H, W = norm.shape
         # Ramer-Douglas-Peucker tolerance — drops near-collinear points
-        # while preserving curvature. 0.75 px keeps the eye-visible shape
-        # while substantially reducing vertex counts on typical PySOAX
-        # centerlines (every-pixel sampling → handful of inflection points).
-        polyline_eps_px = 0.75
+        # while preserving curvature. Bumped from 0.75 → 2.0 px after user
+        # feedback that polylines were too dense / visually cluttered on
+        # MT projects (every-pixel PySOAX sampling produced tens of points
+        # per centerline; 2.0 px keeps the rendered shape essentially
+        # identical at on-screen scales but reduces vertex counts by ~3×
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)